mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -96,6 +96,25 @@ def TilingInterface : OpInterface<"TilingInterface"> {
           return failure();
         }]
       >,
+      InterfaceMethod<
+        /*desc=*/[{
+          Method to return the position of iteration domain tile computed by the
+          tiled operation.
+        }],
+        /*retType=*/"LogicalResult",
+        /*methodName=*/"getIterationDomainTileFromOperandTile",
+        /*args=*/(ins
+          "OpBuilder &":$b,
+          "unsigned":$operandNumber,
+          "ArrayRef<OpFoldResult> ":$offsets,
+          "ArrayRef<OpFoldResult> ":$sizes,
+          "SmallVector<OpFoldResult> &":$iterDomainOffsets,
```

Comment:
I see that this is cargo-culted from the method above, but we should be using `SmallVectorImpl` to avoid hardcoding the implicit number of stack elements. `SmallVector` is in fact `SmallVector<6>` in this case.

Reply:
Agree. I changed parameter type to SmallVectorImpl as well as the code I  cargo-culted from.